package.json

```diff
@@ -1,53 +1,55 @@
 {
-    "name": "oidc-client",
-    "version": "1.0.0-beta.7",
-    "description": "OpenID Connect (OIDC) & OAuth2 client library",
-    "main": "lib/oidc-client.js",
-    "scripts": {
-        "build:lib": "webpack -p --config webpack.lib.js",
-        "build:dist": "webpack -p --config webpack.dist.js",
-        "prebuild": "npm run build:lib",
-        "build": "npm run build:dist",
-        "start": "node sample/server.js",
-        "test": "mocha --compilers js:babel-register test/unit/*.spec.js"
+  "name": "oidc-client",
+  "version": "1.0.0-beta.7",
+  "description": "OpenID Connect (OIDC) & OAuth2 client library",
+  "main": "lib/oidc-client.min.js",
+  "scripts": {
+    "build:lib": "webpack -p --config webpack.lib.js",
+    "build:dist": "webpack -p --config webpack.dist.js",
+    "prepublish": "npm run build",
+    "build": "gulp build",
+    "start": "node sample/server.js",
+    "test": "mocha --compilers js:babel-register test/unit/*.spec.js"
+  },
```

Comment:
Added `prepublish` so that you'll never have to worry about forgetting to run `build` before publish (just a convenience). Changed `build` script to use `gulp`.